packages/venia-concept/src/RootComponents/Category/categoryContent.js

```diff
@@ -21,13 +45,26 @@ class CategoryContent extends Component {
                         }}
                     />
                     <div className={classes.categoryTitle}>{categoryTitle}</div>
+                    <div className={classes.headerButtons}>
+                        <button
+                            onClick={openDrawer}
+                            className={classes.filterButton}
+                        >
+                            Filter
+                        </button>
+                    </div>
                 </h1>
                 <section className={classes.gallery}>
                     <Gallery data={items} title={title} pageSize={pageSize} />
                 </section>
                 <div className={classes.pagination}>
                     <Pagination pageControl={pageControl} />
                 </div>
+                <FilterModal
```

Comment:
And maybe instead of worrying about returning null within the modal component we can conditionally render here as well?

Reply:
Fixed in latest commit.